### DISCUSSION TITLE
Old-school Xfce lives on! - Xfce-4.4 LXQt theme

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2380] [Show and tell]
A long time ago, before Xfce went to GTK3 and subsequently lost its signature themes, it used to be my favorite desktop. With the toolkit switch, its emphasis on client-side decorations, and its stagnating development, I lost interest, but have always fondly remembered the way things used to be.

This is a WIP (~99% done) theme that aims to replicate the old Xfce-4.4 GTK2 theme in LXQt. It just needs a little tweaking to do with indicators and such. I've also created an Openbox theme which is surprisingly close to the "Default-4.4" theme meant to go along with it.

![LXQt-Xfce44](https://user-images.githubusercontent.com/67122280/219044874-661fc0ff-db8a-46d9-8ead-857afd3fa965.png)

I also have Xfce-4.0 and Xfce-curve more or less done. Probably I won't be remaking the entire set, but Xfce-4.2 and maybe a few others are on the radar.

**Comment by AzumaHazuki** [comment]
Repository for these themes now exists at https://github.com/AzumaHazuki/lxqt-themes-xfce and each includes a palette and Openbox theme :)

**Reply by stefonarch** [reply]
You can include a link here: https://github.com/lxqt/lxqt/wiki/Theming

**Reply by edwardp3** [reply]
Have these working now. Thank you.

**Comment by eriefisher** [comment]
This is awesome. I loved the old XFCE. It's a shame the way development is going these days. It's just Gnome basically.

**Reply by AzumaHazuki** [reply]
Yeah, Xfce was my first really long-term desktop...I started on Linux right after the 2.4-->2.6 migration (remember that?) when Gnome was at about version 2.4, and used that until Xfce hit 4.4 and introduced the Thunar file manager. After they migrated to GTK3, I switched to LXDE for a while, but that wasn't a permanent solution. I'm hoping this is, since LXQt themes aren't tied to a specific toolkit the way GTK themes are...

**Reply by eriefisher** [reply]
About the same timing for me. What got me started was the first Knoppix cd from Linux magazine I think it was. I eventually ended up on Ubuntu 4.04. I hated GNOME and went to KDE but it was too much and finally stayed with XFCE4 until current but dropped buntu for Debian. Before thunar was XFM I think it was called. It also started with Sawfish for the window manager. Some cool stuff back then. Anyway I love the look you came up with.
